samza-core/src/main/java/org/apache/samza/clustermanager/ContainerManager.java

```diff
@@ -391,6 +402,30 @@ public void registerContainerPlacementAction(ContainerPlacementRequestMessage re
       return;
     }
 
+    /*
+     * When destination host is {@code FORCE_RESTART_LAST_SEEN} its treated as eqvivalent to kill -9 operation for the container
```

Comment:
small thing: 
I think its equivalent to a "kill"  (i.e., sigstop), not "kill -9" (sigkill). 
Could you verify using the yarn docs